build_artifacts/v1/v1.4/v1.4.0/cpu.additional_packages_env.in

```diff
@@ -0,0 +1,3 @@
+conda-forge::langchain[version='>=0.0.352']
+conda-forge::fastapi[version='>=0.105.0']
+conda-forge::uvicorn[version='>=0.25.0']
```

Comment:
Are the lower bounds strictly necessary? If not, it'd be great if you could remove them.

Reply:
I was under the impression that this was the norm based on the previous release. I am happy to remove those if it's okay, but since we have a release cadence, do these lower bounds cause any issues?